src/Browscap/Data/Division.php

```diff
@@ -65,8 +65,17 @@ public function __construct(
         $this->userAgents = $userAgents;
         $this->lite       = $lite;
         $this->standard   = $standard;
-        $this->versions   = $versions;
         $this->fileName   = $fileName;
+
+        foreach ($versions as $version) {
```

Comment:
We have a Json Schema for Division files in `/schema/divisions.json` to validate our resource files.
Please search for the `versions` property and replace it with this:

```json
"versions": {
      "type": "array",
      "items": {
        "anyOf": [
          {
            "type": "string"
          },
          {
            "type": "integer"
          },
          {
            "type": "object",
            "properties": {
              "start": {
                "type": "integer"
              },
              "end": {
                "type": "integer"
              }
            },
            "required": [
              "start",
              "end"
            ]
          }
        ]
      }
    }
```